cmd/run/run.go

```diff
@@ -389,6 +390,8 @@ func (s *ServerContext) Run(ctx context.Context, config *serverconfig.Config) er
 				requestid.NewStreamingInterceptor(),
 				validator.StreamServerInterceptor(),
 				grpc_ctxtags.StreamServerInterceptor(),
+				recovery.StreamPanicInterceptor(s.Logger),
```

Comment:
Panic recovery middleware should be the first in the call chain. Please move this up above `requestid`.

Also, you need to add it to the unary middleware chain above as well.

Reply:
Done